### PR TITLE
add process monitoring

### PR DESCRIPTION
**Pull request by megastef**

### CODE REVIEW

File: docs/monitoring/processes.md [hunk 1]
```diff
@@ -1,12 +1,87 @@
...
-- overview
-- how it works
-- what it does
-- what it solves 
-- how it helps with observability/troubleshooting/debugging/etc…
+The process information is available in the Monitoring *Infrastructure* section of your Sematext Cloud account - your central place for hosts, virtual machines, containers, and process information. 
+
+### Process View
+
+The process view provides you similar information as the Linux `top` command, but for your entire infrastructure:
```
Comment (adnanrahic) [inline docs/monitoring/processes.md]: The process view provides you similar information as the Linux `top` command, but for your entire infrastructure: 
->
The process view provides similar information as the Linux `top` command. It shows:

File: docs/monitoring/processes.md [hunk 2]
```diff
@@ -1,12 +1,87 @@
...
+- Grouping metrics by hosts, process names, containers, etc. 
+- Filtering by tags
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="../../images/monitoring/sematext-infra-processes.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+
+The real-time view of the top N processes contains a time series chart, heatmap, and a list with the process details help you to identify the processes with the highest resource usage.
```
Comment (adnanrahic) [inline docs/monitoring/processes.md]: The real-time view of the top N processes contains a time series chart, heatmap, and a list with the process details help you to identify the processes with the highest resource usage. 
=>
The real-time view of the top N processes contains a time-series chart, a heatmap, and a list with the process details that help you identify processes with the highest resource usage.

File: docs/monitoring/processes.md [hunk 3]
```diff
@@ -1,12 +1,87 @@
...
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="../../images/monitoring/sematext-infra-processes.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+
+The real-time view of the top N processes contains a time series chart, heatmap, and a list with the process details help you to identify the processes with the highest resource usage. 
+
+Expanding a row in the process list gives detailed information about the processes including:
```
Comment (adnanrahic) [inline docs/monitoring/processes.md]: Expanding a row in the process list gives detailed information about the processes including:
->
Expanding a row in the process list shows detailed information about the processes:

**Comment by adnanrahic** [comment]
Go ahead and merge :)

File: docs/monitoring/processes.md (hunk 2 above)
Comment (otisg) [reply docs/monitoring/processes.md]: Btw @megastef & @adnanrahic we call that tile map, not heat map.  Heat map has X & Y axis.

File: docs/monitoring/processes.md (hunk 3 above)
Comment (otisg) [reply docs/monitoring/processes.md]: about the processes => about processes
or
about the processes => about the process

Comment (adnanrahic) [reply docs/monitoring/processes.md]: _about the process_

**Comment by adnanrahic** [comment]
@megastef fix these two last comments from above and then merge. :ok_hand:

**Pull request merged**